File: day128/day128-logging-libraries/java-lib/src/main/java/com/distributedlogs/LogLevel.java
package com.distributedlogs;

public enum LogLevel {
    DEBUG("DEBUG"),
    INFO("INFO"),
    WARNING("WARNING"),
    ERROR("ERROR"),
    CRITICAL("CRITICAL");
    
    private final String value;
    
    LogLevel(String value) {
        this.value = value;
    }
    
    public String getValue() {
        return value;
    }
}

File: day128/day128-logging-libraries/dotnet-lib/LogLevel.cs
namespace DistributedLogging
{
    public enum LogLevel
    {
        Debug,
        Info,
        Warning,
        Error,
        Critical
    }
    
    public static class LogLevelExtensions
    {
        public static string ToStringValue(this LogLevel level)
        {
            return level switch
            {
                LogLevel.Debug => "DEBUG",
                LogLevel.Info => "INFO",
                LogLevel.Warning => "WARNING",
                LogLevel.Error => "ERROR",
                LogLevel.Critical => "CRITICAL",
                _ => "INFO"
            };
        }
    }
}
